components/monitoring/prometheus/base/external-secrets/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
in development, the external-secrets-opeator isn't deployed (at least for now), so I suggest not putting crds it consumes in base.